lib/cdo/shared_constants/pd/teacher2021_application_constants.rb

```diff
@@ -49,7 +49,11 @@ def self.clean_multiline(string)
           how_heard: 'How did you hear about this program?'
         }
       ),
-      section_2_choose_your_program: {
+      teaching_background:
+        BASE_PAGE_LABELS[:section_2_your_school].slice(
```

Comment:
Ah, so we have some mentions of section_X further upstream?  It's fine to keep this change constrained but also interested to ponder how far we can take the numberless principle :)

Reply:
I agree! I'm going to fix that as part of an upcoming change to simplify how our constants files are structured.